src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -263,9 +263,11 @@ function assertValidProps(component, props) {
   }
   invariant(
     props.style == null || typeof props.style === 'object',
-    'The `style` prop expects a mapping from style properties to values, ' +
+    ' A component inside its render method has incorrect inline style syntax.' +
+     ' The `style` prop expects a mapping from style properties to values, ' +
     'not a string. For example, style={{marginRight: spacing + \'em\'}} when ' +
-    'using JSX.'
+    'using JSX. %s',
```

Comment:
Don't put a space before `%s`. If you look at what `getDeclarationErrorAddendum` is doing, it prefixes with a space.